packages/foreman/nodejs-patternfly/nodejs-patternfly.spec

```diff
@@ -5,7 +5,7 @@
 
```

Comment:
Why is the filesize for this increasing by 120%? Shouldn't it just be a trivial rebuild?

Reply:
I assume you mean the registry tarball. Any bundled nodejs module has to have a tarball generated based on the Nodejs 10 structure since it changed from nodejs 6.

Reply:
Still odd that it's so much larger.